deploy/olm-catalog/apicast-community-operator/apicast-community-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -25,15 +25,15 @@ spec:
       name: apicasts.apps.3scale.net
       version: v1alpha1
   description: |
-    The APIcast Operator creates and maintains the Red Hat 3scale API Gateway in several deployment configurations.
+    The APIcast Community Operator creates and maintains the Red Hat 3scale API Gateway in several deployment configurations.
```

Comment:
I would not make differentiation of community and productized here, `APIcast Operator` is just fine

Reply:
:+1: